README.md

```diff
@@ -1,22 +1,21 @@
-# Google Engineering Practices Documentation
+# Shpock Backend Engineering Practices Documentation
```

Comment:
I can't add a comment at the License sections, but do you think as per the CC-By 3.0 terms we should maybe attribute the original work?